src/topics/inputs.md

```diff
@@ -298,4 +370,3 @@ a default value if none was provided in the command line or job input file.
 % - Several ways of defining inputs/arguments to tools and workflows - https://github.com/common-workflow-language/user_guide/issues/33
 % - Using an input output in another input - https://github.com/common-workflow-language/user_guide/issues/90
 % - How to use linkMerge - https://github.com/common-workflow-language/user_guide/issues/117 (or maybe move to Advanced?)
-% - Secondary files - https://github.com/common-workflow-language/common-workflow-language/issues/270
```

Comment:
Thanks for keeping the list updated! :medal_sports:

Reply:
> Thanks for keeping the list updated! 🏅

Thank you for the review!